src/com/rcv/Logger.java

```diff
@@ -55,7 +57,10 @@ static void setup() throws IOException {
     // formatter specifies how logging output lines should appear
     LogFormatter formatter = new LogFormatter();
     // fileHandler writes formatted strings to file
-    FileHandler fileHandler = new FileHandler(logPath, true);
+    FileHandler fileHandler = new FileHandler(logPath,
+        TABULATION_LOG_FILE_MAX_SIZE_BYTES,
+        1,
```

Comment:
Are you sure this does what you want? I'm wondering what will happen if you start the app when the file is slightly under the limit. It would be bad if it logged to the file until it hit the limit, then started over on the same file and continued logging. The user would end up not being able to access the lines that were logged before the roll happened.

Maybe a safer approach is to set a limit of 2 and make the per-file size half as big?

Reply:
Yeah that's a better idea.